SPIFlashFileSystem.class.nut

```diff
@@ -123,10 +123,11 @@ class SPIFlashFileSystem {
         if (!_fat.fileExists(fname)) throw ERR_FILE_NOT_FOUND;
         if (isFileOpen(fname)) throw ERR_OPEN_FILE;
 
-        _enable();
-
         // Build a blob to zero out the file
         local zeros = blob(SPIFLASHFILESYSTEM_HEADER_SIZE + SPIFLASHFILESYSTEM_TIMESTAMP_SIZE + SPIFLASHFILESYSTEM_MAX_FNAME_SIZE + 1);
+
```

Comment:
There is no special reason for that except that _enable() method allocate "potentially critical" resource AND blob creation does not required that resource.

So, please let me know if it could be a problem with merge of this change on the main branch ?